rest-service/dev-requirements.txt

```diff
@@ -1,4 +1,4 @@
-git+https://github.com/cloudify-cosmo/cloudify-common@master#egg=cloudify-common==4.5.5.dev1
+git+https://github.com/cloudify-cosmo/cloudify-common@CY-693#egg=cloudify-common==4.5.5.dev1
```

Comment:
This shouldn't be in a PR

Reply:
Need here so the unit tests can be run, because this change depends on common change